lib/temporal/activity/context.rb

```diff
@@ -39,8 +41,29 @@ def async_token
         )
       end
 
+      # Returns true if the activity has been canceled directly by its workflow or indirectly by
+      # its workflow being terminated. This will only be set following a call to heartbeat that is
+      # not throttled.
+      def cancel_requested
```

Comment:
nit: let's go with `cancel_requested?` to be consistent with other query-type methods

Reply:
Looks like we may want to call thins `cancelled?` instead based on the sdk-ruby API?

Reply:
@antstorm This is just moving an existing method from an attr_reader to a proper method so that I can more clearly put a comment on it. I'm happy to alias this as `cancelled?` and mark the existing method as deprecated if that's the direction you want to move instead.